### PR TITLE
generate certificates directly in the volume Mount to avoid getting p…

### PR DESCRIPTION
**Pull request by itmwiw**
…ermissions issues on openshift with restricted scc

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you updated the readme?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss open a ticket first**

### CODE REVIEW

File: templates/statefullset.yaml [hunk 1]
```diff
@@ -49,10 +49,8 @@ spec:
             - -c
             - |
               {{- if not .Values.customTLS.enabled}}
-              openssl req -x509 -newkey rsa:4096 -nodes -subj '/CN={{ .Values.global.ldapDomain }}' -keyout tls.key -out tls.crt -days 365
-              cp tls.crt ca.crt
-              cp *.crt /tmp-certs/
-              cp tls.key /tmp-certs/
+              openssl req -x509 -newkey rsa:4096 -nodes -subj '/CN=example.org' -keyout /tmp-certs/tls.key -out /tmp-certs/tls.crt -days 365
```
Comment (jp-gouin) [inline templates/statefullset.yaml:52]: Subject should be `-subj '/CN={{ .Values.global.ldapDomain }}'` and not hardcoded to `'/CN=example.org'`

**Comment by jp-gouin** [comment]
thanks @itmwiw for the PR :)

File: templates/statefullset.yaml (hunk 1 above)
Comment (itmwiw) [reply templates/statefullset.yaml]: you're welcome. Done.

**Comment by jp-gouin** [comment]
Corrected in latest release `v4.1.0`

**Pull request closed without merging**